presentation/src/main/java/com/architect/coders/mu8/characters/CharactersAdapter.kt

```diff
@@ -6,6 +6,8 @@ import android.widget.ImageView
 import android.widget.TextView
```

Comment:
Please review the imports on all classes, this one in particular has several unused imports.

Reply:
ok